slashCommands/vote.ts

```diff
@@ -29,6 +29,13 @@ export default {
                 description: "Time the vote is running in seconds.",
                 required: false,
                 minValue: 0
+            },
+            {
+                type: ApplicationCommandOptionType.Integer,
```

Comment:
There is "ApplicationCommandOptionType.Role" (at least according to the documentation) - Wouldn't that be fitting here, then we could avoid all the role-integer stuff